contrib/systemd/system/podman.service

```diff
@@ -1,7 +1,8 @@
 [Unit]
 Description=Podman API Service
 Requires=podman.socket
-After=podman.socket
+Wants=network.target
+After=podman.socket network-online.target
```

Comment:
As former systemd developer/maintainer I recommend to not do that. podman.service does *not* need networking at startup, only for some operations -- but that shouldn't affect the boot order. network-online.target is a rather terrible and brittle hack, see `man systemd.special`.

`Wants=network.target` is also unnecessary here. Your unit shouldn't activate that -- at most you'd have `After=network.target`, but that's not even required as podman does not require the network to shut down cleanly.